taca/nanopore/promethion_transfer.py

```diff
@@ -12,11 +13,14 @@ def main(args):
     """Find promethion runs and transfer them to storage. 
     Archives the run when the transfer is complete."""
     data_dir = args.source_dir
+    project_pattern = re.compile("^P\d{5}$")  # Runs started manually (project ID)
```

Comment:
Just to nag on something, I think we can expand this to include more potential p-numbers.
```suggestion
    project_pattern = re.compile("^P\d{4,6}$")  # Runs started manually (project ID)
```